packages/node/src/types.ts

```diff
@@ -66,3 +66,9 @@ export interface TakeActionMessage extends NodeMessage {
     appInstanceId: string;
   };
 }
+
+export interface RejectProposalMessage extends NodeMessage {
+  data: {
```

Comment:
There should probably be another field in the data section (maybe: `reason` or something) so the rejecting Node can indicate why they rejected the proposal.